openshift/machine-config-operator
# Bug 1990012: Update controller config openapi schema

**- What I did**

I used controller-gen to regenerate the openAPIV3Schema section of this CRD.
This means that it is now up to date and includes all of the latest fields from the objects that are embedded within it.

**- How to verify it**
Find a field such as `.spec.infra.status.platformStatus.azure.cloudName` which is not currently present in the CRD schema. Observe how this exists in the `infrastructure` resource, but does not exist in the embedded infra in the `controllerconfig` in clusters today. Try to add this field into the `controllerconfig`, it will be silently dropped by the API server.
Upgrade to this patch, observe how the field now exists and is copied from the main `infrastructure` resource.

**- Description for the changelog**


Update OpenAPI schema for `controllerconfig` CRD
